test/Constraints/members.swift

```diff
@@ -590,3 +590,36 @@ do {
   throw SR_2193_Error.Boom
 } catch let e as SR_2193_Error.Boom { // expected-error {{enum element 'Boom' is not a member type of 'SR_2193_Error'}}
 }
+
+// rdar://problem/25341015
+extension Sequence {
+  func r25341015_1() -> Int {
+    return max(1, 2) // expected-error {{type member 'max' shadows global function 'max' in module 'Swift', which cannot be called using implicit 'self.'}} {{12-12=Swift.}}
```

Comment:
Actually, I think I was wrong about this.

When a global function conflicts with an instance member, `which cannot be called using implicit 'self.'` is a bit confusing.

But when a *static member* conflicts with an instance member, the message is correct. So, maybe you can just disregard what I said above...